templates/Shop/TaxonProductsSearch/content/body/main/sorting.html.twig

```diff
@@ -0,0 +1,23 @@
+{% if hookable_metadata.context.products is defined and hookable_metadata.context.products is not empty and hookable_metadata.context.products.nbResults is defined and hookable_metadata.context.products.nbResults > 0 %}
+
+    {% set route = app.request.attributes.get('_route') %}
+    {% set route_parameters = app.request.query.all|unset_elements(['order_by', 'sort', 'page']) %}
+
+    {% if app.request.query.all()['order_by'] is not defined or app.request.query.all()['order_by'] is empty %}
```

Comment:
I wonder whether we couldn't put this into twig function, to not to check request on templates.

Reply:
Good